core/tests/protractor_utils/ExplorationPlayerPage.js

```diff
@@ -80,6 +80,49 @@ var ExplorationPlayerPage = function() {
   var suggestionPopupLink =
     element(by.css('.protractor-test-exploration-suggestion-popup-link'));
 
+  var audioBar = element(
+    by.css('.protractor-test-audio-bar'));
+  var audioSelector = element(
+    by.css('.protractor-test-audio-lang-select'));
+  var playButton = element(
+    by.css('.protractor-test-play-circle'));
+  var pauseButton = element(
+    by.css('.protractor-test-pause-circle'));
+
+  this.clickAudioBar = async function() {
+    await waitFor.elementToBeClickable(audioBar,
+      'Audio bar taking too long to be clickable');
+    await audioBar.click();
+  };
+
+  this.playAudio = async function() {
+    await waitFor.elementToBeClickable(playButton,
+      'Play button taking too long to be clickable');
+    await playButton.click();
+  };
+
+  this.expectAudioToBePlaying = async function() {
+    expect(pauseButton.isPresent()).toBeTruthy();
```

Comment:
`await playButton.isPresent()`?

Reply:
When audio is playing, I'd expect the pause button to be showing. That's why I used `expect(pauseButton.isPresent()).toBeTruthy()` to assert that the pause button is showing and audio is playing.

Reply:
done!